<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Montserrat</family>
    <bold>false</bold>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>EEG Viewer</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QFrame" name="frSerialConnection">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>611</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QGroupBox" name="gBSerialConnection">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>191</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Serial</string>
     </property>
     <widget class="QFrame" name="frSerialSend">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>421</width>
        <height>151</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-1</y>
         <width>421</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLoutSerialSend">
        <item>
         <layout class="QHBoxLayout" name="hLoutSerialSend1">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lnInputSend1"/>
          </item>
          <item>
           <widget class="QPushButton" name="pBtnInputSend1">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hLoutSerialSend2">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lnInputSend2"/>
          </item>
          <item>
           <widget class="QPushButton" name="pBtnInputSend_2">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hLoutSerialSend3">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lnInputSend_3"/>
          </item>
          <item>
           <widget class="QPushButton" name="pBtnInputSend3">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frSerialConnect">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>151</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>160</width>
         <height>155</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLoutSerialConnect">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="lblSerialName">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cBSerialName"/>
        </item>
        <item>
         <widget class="QLabel" name="lblSerialBaud">
          <property name="text">
           <string>Baud</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cBSerialBaud"/>
        </item>
        <item>
         <widget class="QPushButton" name="pBtnSerialConnect">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <bold>true</bold>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 85, 0)</string>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frTransceivedData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QGroupBox" name="gBTransceivedData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>491</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Received/Sent Data</string>
     </property>
     <widget class="QTextBrowser" name="tBrTransceivedData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>471</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
